autode/opt/optimisers/trm.py

```diff
@@ -339,7 +339,9 @@ def cart_step_length_error(int_size):
 
         # The value of shift parameter lambda must lie within (-infinity, first_b)
         # Need to find the roots of the 1D function cart_step_length_error
-        int_step_size = self.alpha  # starting guess, so ok to use cartesian
+        int_step_size = float(
```

Comment:
worth making `int_step_size` a `Distance` too?

Reply:
@t-young31 I think the internal step size should have units of angstrom - radians or something similar as it contains both distances and angles. So i think it's not needed to make it Distance.

Reply:
good point – didn't spot it was a step in internals